### ISSUE TITLE
Data Management API Endpoint in Control Plane Chart wrong?

### ISSUE THREAD

**Issue by florianrusch-zf** [catenax-ng/product-edc#34]
Hi all together,

isn't the path of the data mangement api endpoint in the ingress wrong? Shouldn't it be `/api/v1/data`?
https://github.com/catenax-ng/product-edc/blob/dda4589979c58f86b46a448fe6e1d20979a5ab75/deployment/helm/edc-controlplane/values.yaml#L93-L97

\cc @denisneuling @DominikPinsel

**Comment by DominikPinsel** [comment]
As the EDC is now all paths must be configured explicitly. If no path is configured the data management api will be available under the `default` context port and path, configured in `web.http.default.port` and `web.http.default.path`.

As far as I know the `/api/v1/data` is used in the EDC repository out of habit, but there are no guidelines or best practices how these API paths should look like. It was probably named like this, to have the path somewhat looking like the IDS path (which was not configurable in the past).

The `/data/` path was chosen to have a consistent schema for the paths and to keep them short.  Therefore I'd say using `/data` here should be fine.

But please correct me if I'm wrong somewhere.

<sub>Dominik Pinsel <dominik.pinsel@daimler.com>, Mercedes-Benz Tech Innovation GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md) </sub>

**Comment by florianrusch-zf** [comment]
Of course, but then this has also to be considered in the configurations. In my eyes it would be better for "demo/onboarding" purposes to use the "default" (also if it is not the default 😄). The default is used in the edc documentation and also CatenaX-at-home uses the "default". I think this only lead to confusion for users if we now use a different path here.

**Comment by DominikPinsel** [comment]
While it's definitely an idea to add the `/api` prefix to all the paths, I'm not sure about the `/v1` part. It's not really a version 1 and we would need to update the configuration every time the EDC adds breaking changes to the API (if we do it right).

**Comment by florianrusch-zf** [comment]
I agree with you, that the current version of the data management api is not really a version 1. On the other hand, it usually makes sense to have some kind of api versioning, especially if you make it available to others.

The edc is "just" a framework, not a runnable application, which means that any hosting person can add additional api endpoints (as we do in this repo, for example). So this person should also take care of the api versioning.

If we decide not to provide api-versioning, then that's fine for me. But in my opinion, api versioning is not wrong in most cases. But of course we can also say that we don't want to provide this as a default for now and introduce it later.

**Comment by DominikPinsel** [comment]
My concern is that if we write v1 in the path and there are breaking changes in the API, users can come up to us and we have to explain the breaking changes in v1. 
As I'm sure there will be breaking changes in the EDC, I'd like to make sure we can easily refer to the EDC and don't need to explain ourselves.
